src/vehicle.h

```diff
@@ -1113,6 +1113,9 @@ class vehicle
             bool fullsize = false, bool verbose = false, bool desc = false,
             bool isHorizontal = false );
 
+        // Vehicle speed gauge
+        void print_speed_gauge( const catacurses::window &win, const point &, unsigned int spaceing = 0 );
```

Comment:
```suggestion
        void print_speed_gauge( const catacurses::window &win, const point &, unsigned int spacing = 0 );
```

Reply:
Yeah, stupid me amended, can't push fix, till undo that, but saw clang-tidy failing and jumped on it.